spec/exe/floe_spec.rb

```diff
@@ -1,14 +1,122 @@
-RSpec.describe "exe/floe" do
+RSpec.describe "exe/floe", :slow => true do
```

Comment:
Thought this might be useful, since these tests are kinda slow, so they can be ignored in local dev with `--tag ~slow`